### PR TITLE
`run` command

### PR DESCRIPTION
**Pull request by mih**

### CODE REVIEW

File: datalad_container/containers_run.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+@build_doc
+# all commands must be derived from Interface
+class ContainersRun(Interface):
+    # first docstring line is used a short description in the cmdline help
+    # the rest is put in the verbose help and manpage
+    """Drop-in replacement of 'run' to perform containerized command execution
+
+    Container(s) need to be configured beforehand (see containers-add).
+    If only one container is known, it will be selected automatically,
+    otherwise a specific container has to be selected.
```
Comment (kyleam) [inline datalad_container/containers_run.py]: minor, but I think replacing the second "selected" with "specified" would be clearer.

File: datalad_container/containers_run.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+_run_params = dict(
+    Run._params_,
+    container_name=Parameter(
+        args=('-n', '--container-name',),
+        metavar="NAME",
+        doc="""Specify the name of a known container to use for execution,
+        in case multiple containers are configured."""),
+    # have to override the cmd parameter, because in `run` it says
+    # SHELL COMMAND but with a container it could be something totally
```
Comment (kyleam) [inline datalad_container/containers_run.py]: I don't know that the SHELL in run's `metavar='SHELL COMMAND'` really adds that much.  Perhaps we should just change it to `metavar='COMMAND'` there and make sure Run's class docstring mentions explicitly that it's a shell command.

Comment (mih) [reply datalad_container/containers_run.py]: That would be a solution.

File: datalad_container/containers_run.py (hunk 1 above)
Comment (mih) [reply datalad_container/containers_run.py]: Done

**Pull request merged**